doc/dox_comments/header_files/ssl.h

```diff
@@ -4538,6 +4629,66 @@ WOLFSSL_API WOLFSSL_STACK* wolfSSL_X509_STORE_CTX_get_chain(
 WOLFSSL_API int wolfSSL_X509_STORE_set_flags(WOLFSSL_X509_STORE* store,
                                                             unsigned long flag);
 
+/*!
+    \ingroup CertsKeys
+
+    \brief This function returns the value stored in the sigOID
```

Comment:
I don't think this is correct.  This should be returning the certificate "not before" validity encoded as a byte array, right?